package.json

```diff
@@ -6,14 +6,18 @@
     "date-fns": "^4.1.0",
     "dotenv": "^16.4.5",
     "googleapis": "^144.0.0",
-    "ts-node": "^10.9.2"
-  },
-  "scripts": {
-    "full-sync": "rm -rf HaTo-* && npx ts-node index.ts",
-    "quick-sync": "rm -rf HaTo-* && npx ts-node index.ts --no-bank-sync"
+    "ts-node": "^10.9.2",
+    "rimraf": "^5.0.0"
```

Comment:
```suggestion
```
I see that this would be required for non-unix users who run the script outside of linux. Though the `clean` script is custom tailored to my setup (see commend below).